.github/workflows/mingw-w64.yml

```diff
@@ -0,0 +1,65 @@
+name: mingw-w64
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    name: Build
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        with-patch:
+          - true
+          - false
+    env:
+      MINGW_PACKAGE_PREFIX: mingw-w64-ucrt-x86_64
+      MSYSTEM: UCRT64
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: dependencies
+        shell: c:\msys64\usr\bin\bash.exe -e -l {0}
+        run: |
+          pacman \
+            --needed \
+            --noconfirm \
+            --sync \
+            ${MINGW_PACKAGE_PREFIX}-cc \
+            ${MINGW_PACKAGE_PREFIX}-cmake \
+            ${MINGW_PACKAGE_PREFIX}-curl \
+            ${MINGW_PACKAGE_PREFIX}-ninja \
+            ${MINGW_PACKAGE_PREFIX}-openssl \
+            ${MINGW_PACKAGE_PREFIX}-zlib
```

Comment:
`${MINGW_PACKAGE_PREFIX}` is `mingw-w64-ucrt-x86_64`.

It means:

* `mingw-w64-ucrt`: We use mingw-w64 GCC with universal C runtime
* `x86_64`: We use Intel/AMD 64bit CPU (We can use `aarch64` for Arm 64bit CPU)